2016-2017_Starstruck/Swerve Driver Proto/include/SwerveLib.h

```diff
@@ -96,6 +96,9 @@
 	 * Changes the active drive configuration
 	 */
 	 void setDriveConfig(DriveConfiguration config);
-		void setDriveConfigById(int id);
+	 void setDriveConfigById(int id);
+
+	void pulseMotor(unsigned char x);
```

Comment:
Could you add comments above that say what these do like we have for the rest of them?